code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -434,16 +426,14 @@
 	R.touch_turf(target)
 
 
-/datum/reagents/proc/trans_to_obj(var/turf/target, var/amount = 1, var/multiplier = 1, var/copy = 0) // Objects may or may not; if they do, it's probably a beaker or something and we need to transfer properly; otherwise, just touch.
-	if(!target || !target.simulated)
+/datum/reagents/proc/trans_to_obj(var/obj/target, var/amount = 1, var/multiplier = 1, var/copy = 0) // Objects may or may not; if they do, it's probably a beaker or something and we need to transfer properly; otherwise, just touch.
+	if(!target)
```

Comment:
I don't think this should be modified. Pretty sure this causes unusual behaviour. Did you test this?